scripts/caput-pipeline

```diff
@@ -70,13 +70,32 @@ def queue(configfile, nosubmit=False):
         raise Exception('Job time is required.')
     clusterconf['time'] = conf['time']
 
+    if 'cluster' not in conf:
```

Comment:
Parameter shouldn't be mandatory as this will break all previous YAML files. I'd suggest pick a default, and maybe emit a warning if it wasn't set. Alternatively (and probably preferably), you could check the system for the presence of `sbatch` or `qsub` to decide whether it's a PBS or SLURM system.